pkgs/by-name/ce/celeste64/package.nix

```diff
@@ -0,0 +1,68 @@
+{
+  lib,
+  buildDotnetModule,
+  dotnetCorePackages,
+  fetchFromGitHub,
+  SDL2,
+  libGL,
+  fmodex,
+  libdecor,
+  libxkbcommon,
+  wayland,
+  systemd,
+  alsa-lib,
+  libselinux,
+  withWayland ? true,
+  withSELinux ? false,
+  ...
+}: let
+  rev = "62417823a30582e8561075fe0f7d4b398aedc353";
+  version = builtins.substring 0 6 rev;
+in
+  buildDotnetModule {
+    pname = "celeste64";
+    version = "2024.01.31-${version}";
+
+    src = fetchFromGitHub {
+      repo = "Celeste64";
+      owner = "ExOK";
+      sha256 = "sha256-S9KOtCrJSsIQxvZWjiHUYEiM9/TbO0q6vSDPwJfyW3I=";
+      inherit rev;
+    };
+    projectFile = "Celeste64.csproj";
+    dotnet-sdk = dotnetCorePackages.sdk_8_0;
+    dotnet-runtime = dotnetCorePackages.runtime_8_0;
+    nugetDeps = ./deps.nix;
+    strictDeps = true;
+    runtimeDeps =
+      [
+        wayland
+        libdecor
+        libGL
+        SDL2
+        fmodex
+        libxkbcommon
+        systemd
+        alsa-lib
+        libselinux
+      ]
+      ++ lib.optionals withWayland [ wayland ]
```

Comment:
Why add it here optionally if you add it unconditionally in line 39 anyway?

Please remove one of these.